cypress/e2e/tests/custom-metrics/issues_tracker.test.ts

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Karishma , just noticed , the test names of other metrics are different.
Please rename this to jira_issue_metrics.test.ts

Reply:
Done